src/test/java/codeu/controller/ActivityFeedServletTest.java

```diff
@@ -0,0 +1,89 @@
+package codeu.controller;
+
+import codeu.model.data.Conversation;
+import codeu.model.data.Message;
+import codeu.model.data.User;
+import codeu.model.store.basic.ConversationStore;
+import codeu.model.store.basic.MessageStore;
+import codeu.model.store.basic.UserStore;
+import java.io.IOException;
+import java.time.Instant;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.UUID;
+import javax.servlet.RequestDispatcher;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.servlet.http.HttpSession;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.ArgumentCaptor;
+import org.mockito.Mockito;
+
+public class ActivityFeedServletTest {
+    private ActivityFeedServlet activityFeedServlet;
+    private HttpServletRequest mockRequest;
+    private HttpSession mockSession;
+    private HttpServletResponse mockResponse;
+    private RequestDispatcher mockRequestDispatcher;
+    private ConversationStore mockConversationStore;
+    private MessageStore mockMessageStore;
+    private UserStore mockUserStore;
+
+    @Before
+    public void setup() {
```

Comment:
same as before - please match existing indentation (only +2 spaces, not 4)

Reply:
Done